runtime/vm/intfunc.c

```diff
@@ -228,6 +228,9 @@ J9InternalVMFunctions J9InternalFunctions = {
 	structuredSignalHandlerVM,
 	addHiddenInstanceField,
 	fieldOffsetsStartDo,
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
```

Comment:
no ifdefs

Reply:
@tajila The struct J9InternalVMFunctions in `j9nonbuilder.h` defined this function pointer, and I ifdef'd it. Do I move that out of ifdef as well?